mlflow_faculty/mlflow_converters.py

```diff
@@ -128,3 +130,12 @@ def mlflow_timestamp_to_datetime_milliseconds(mlflow_timestamp):
 
 def mlflow_timestamp_to_datetime_seconds(mlflow_timestamp):
     return datetime.fromtimestamp(mlflow_timestamp, tz=UTC)
+
+
+def mlflow_viewtype_to_faculty_lifecycle_stage(mlflow_view_type):
+    conversions = {
+        ViewType.ACTIVE_ONLY: FacultyLifecycleStage.ACTIVE,
+        ViewType.DELETED_ONLY: FacultyLifecycleStage.DELETED,
+        ViewType.ALL: None,
+    }
```

Comment:
suggest moving conversions to a module level `LIFECYCLE_STAGE_CONVERSION` as discussed